concent_api/conductor/tasks.py

```diff
@@ -55,23 +56,31 @@ def blender_verification_request(
 
     output_format = output_format.upper()
     assert output_format in BlenderSubtaskDefinition.OutputFormat.__members__.keys()
-
-    # The app creates a new instance of VerificationRequest in the database
-    # and a BlenderSubtaskDefinition instance associated with it.
-    (verification_request, blender_subtask_definition) = store_verification_request_and_blender_subtask_definition(
-        subtask_id=subtask_id,
-        source_package_path=source_package_path,
-        result_package_path=result_package_path,
-        verification_deadline=verification_deadline,
-        output_format=output_format,
-        scene_file=scene_file,
-        blender_crop_script=blender_crop_script,
-    )
-
-    store_frames(
-        blender_subtask_definition=blender_subtask_definition,
-        frame_list=frames,
-    )
+    try:
+        with transaction.atomic(using='storage'):
+            # The app creates a new instance of VerificationRequest in the database
+            # and a BlenderSubtaskDefinition instance associated with it.
+            (verification_request, blender_subtask_definition) = store_verification_request_and_blender_subtask_definition(
+                subtask_id=subtask_id,
+                source_package_path=source_package_path,
+                result_package_path=result_package_path,
+                verification_deadline=verification_deadline,
+                output_format=output_format,
+                scene_file=scene_file,
+                blender_crop_script=blender_crop_script,
+            )
+
+            store_frames(
+                blender_subtask_definition=blender_subtask_definition,
+                frame_list=frames,
+            )
+    except DatabaseError as exception:
+        log_string_message(
```

Comment:
Better to use here `logger.warning`. It should be unified everywhere. If you use in `core.tasks.upload_finished` `logger.warning` then here must be the same.